src/Public/Pages.ps1

```diff
@@ -854,11 +863,33 @@ function ConvertTo-PodeWebPage
             New-PodeWebTab -Name $name -Layouts $form
         })
 
-        $group = [string]::Empty
-        if ($GroupVerbs) {
-            $group = $cmdInfo.Verb
-            if ([string]::IsNullOrWhiteSpace($group)) {
-                $group = '_'
+        switch ($PSCmdlet.ParameterSetName) {
+            'NoGrouping' {
+                $group = [string]::Empty
+                break
```

Comment:
The `break`s aren't really needed in this switch statement, since only a single value is being supplied to the `switch` and not an array.